lib/utils/common.py

```diff
@@ -1649,39 +1649,81 @@ def _create_stop_writes_entry(
 
 @staticmethod
 def _is_stop_writes_cause(
-    usage: int | float, threshold: int | float, stop_writes: str | None = None
+    usage: int | float,
+    threshold: int | float,
+    stop_writes: str | None = None,
+    invert: bool = False,
 ):
     if threshold == 0:
         return False
 
+    if invert:
+        return (
+            True
+            if usage <= threshold
+            and (stop_writes is None or stop_writes.lower() == "true")
+            else False
+        )
     return (
         True
         if usage >= threshold and (stop_writes is None or stop_writes.lower() == "true")
         else False
     )
 
 
+@staticmethod
```

Comment:
Does the staticmethod decorator do anything if this function isn't the member of a class? I don't see the class this is attached to.